l1-contracts/src/core/Rollup.sol

```diff
@@ -462,13 +479,30 @@ contract Rollup is Leonidas, IRollup, ITestRollup {
     _validateHeader(header, _signatures, _digest, _currentTime, _txsEffectsHash, _flags);
   }
 
+  function computeTxsEffectsHash(bytes calldata _body)
```

Comment:
Get my linter to stop yelling at me with respect to ordering of `external` vs `external pure`